go.mod

```diff
@@ -4,16 +4,15 @@ go 1.14
 
 require (
 	github.com/getkin/kin-openapi v0.20.0
-	github.com/golang/protobuf v1.3.3
+	github.com/golang/protobuf v1.4.2
```

Comment:
i think this might give you problems when you try to dynamically load the plugin in to `onos-config` - they would both have to be at the same version